src/platform/nxp/rt/rw61x/BUILD.gn

```diff
@@ -20,6 +20,9 @@ import("//build_overrides/openthread.gni")
 import("${chip_root}/src/platform/device.gni")
 import("${chip_root}/src/platform/nxp/common/ota/ota.gni")
 
+# Include to define nxp_crypto_impl
+import("${chip_root}/src/platform/nxp/crypto/nxp_crypto.gni")
```

Comment:
Can you add same include in https://github.com/project-chip/connectedhomeip/blob/master/src/platform/nxp/mcxw71/BUILD.gn, https://github.com/project-chip/connectedhomeip/blob/master/src/platform/nxp/rt/rt1060/BUILD.gn and https://github.com/project-chip/connectedhomeip/blob/master/src/platform/nxp/rt/rt1170/BUILD.gn